### PR TITLE
fixed XLA flag for simplify fp conversion

### PR DESCRIPTION
**Pull request by kocchop**
removed the deprecated `--xla_gpu_simplify_all_fp_conversions`
replaced with `--xla_allow_excess_precision`

### CODE REVIEW

File: .github/container/test-maxtext.sh [hunk 1]
```diff
@@ -187,7 +187,7 @@ export CUDA_DEVICE_MAX_CONNECTIONS=1
 
 export BASE_XLA_FLAGS=${BASE_XLA_FLAGS:---xla_gpu_enable_latency_hiding_scheduler=true 
                 --xla_gpu_enable_triton_gemm=false
-                --xla_gpu_simplify_all_fp_conversions 
+                --xla_allow_excess_precision
```
Comment (olupton) [inline .github/container/test-maxtext.sh:190]: This seems to be the default, why do we need to set it?

**Review by olupton — CHANGES_REQUESTED** [review]

File: .github/container/test-maxtext.sh (hunk 1 above)
Comment (olupton) [inline .github/container/test-maxtext.sh:190]:
```suggestion
```

Comment (gpupuck) [reply .github/container/test-maxtext.sh:190]: agree

Comment (kocchop) [reply .github/container/test-maxtext.sh:190]: Thanks, will remove it

**Review by olupton — APPROVED** [review]

**Pull request merged**